app/workers/subject_queue_worker.rb

```diff
@@ -15,6 +15,8 @@ def perform(workflow_id, user=nil, limit=SubjectQueue::DEFAULT_LENGTH)
     else
       load_subjects
     end
+  rescue ActiveRecord::RecordNotFound
```

Comment:
That should be handled https://github.com/zooniverse/Panoptes/blob/master/app/models/workflow.rb#L17